src/web/src/components/Posts/Post.tsx

```diff
@@ -315,6 +348,11 @@ const PostComponent = ({ postUrl, currentPost, totalPosts }: Props) => {
     }
   }, [expandHeader]);
 
+  // Listen for click events
+  useEffect(() => {
+    window.document.addEventListener('click', handleZoom);
```

Comment:
It's a good idea to remove the event handler when the component unmouts.  If not, when we navigate to a different page without any posts, say '/about', the event handler is still listening.

Reply:
Yeah somehow I missed this, thanks. I updated it to remove the listener.